src/webextension/manage/components/item-details.js

```diff
@@ -62,27 +61,42 @@ export default class ItemDetails extends React.Component {
         e.preventDefault();
         onSave(this.state);
       }}>
-        <p>
-          <Localized id="item-site">
-            <span>sITe</span>
+        <label>
+          <Localized id="item-details-title">
+            <span>tITLe</span>
           </Localized>
-          <Input type="text" value={this.state.title}
+          <Input type="text" name="title" value={this.state.title}
                  onChange={(e) => this.setState({title: e.target.value})}/>
-        </p>
-        <p>
-          <Localized id="item-username">
+        </label>
+        <label>
+          <Localized id="item-details-origin">
+            <span>oRIGIn</span>
+          </Localized>
+          <Input type="text" name="origin" value={this.state.origin}
+                 onChange={(e) => this.setState({origin: e.target.value})}/>
+        </label>
+        <label>
+          <Localized id="item-details-username">
             <span>uSERNAMe</span>
           </Localized>
-          <Input type="text" value={this.state.username}
+          <Input type="text" name="username" value={this.state.username}
                  onChange={(e) => this.setState({username: e.target.value})}/>
-        </p>
-        <p>
-          <Localized id="item-password">
+        </label>
+        <label>
+          <Localized id="item-details-password">
             <span>pASSWORd</span>
           </Localized>
-          <Input type="text" value={this.state.password}
+          <Input type="text" name="password" value={this.state.password}
                  onChange={(e) => this.setState({password: e.target.value})}/>
-        </p>
+        </label>
+        <label>
+          <Localized id="item-details-notes">
+            <span>nOTEs</span>
+          </Localized>
+          <TextArea name="notes" value={this.state.notes}
+                    onChange={(e) => this.setState({notes: e.target.value})}/>
+        </label>
+        <p></p>
```

Comment:
Nit: An empty `<p></p>` feels weird. Not sure if we should wrap in a `<div>` instead to add some extra padding if we need it.